appspec.yml

```diff
@@ -0,0 +1,30 @@
+version: 0.0
+os: linux
+files:
+  - source: /
```

Comment:
Does this mean that CodeDeploy ll always copy the source files to this destination regardless of stable or nightly? If both the cd pipelines of stable and nightly run together, will there be a race condition where potentially wrong source files are copied to wrong directory in after_install.sh line 10? (e.g. stable files got copied into nightly folder)?

Reply:
I used this condition https://github.com/tlsnotary/tlsn/pull/419/files#diff-27ec5655b90f6efb2697e4cac232cc8ac76273ab346b84891a0f28c81931e171R19-R21
that ensures that only one workflow runs per concurrency group.
If a new run is triggered while another is still in-progress, the in-progress run is canceled to make way for the latest one.
If **cancel-in-progress: false**, the queued workflow will be pending and start right after the in-progress ends.

Reply:
ah nice! but that seems like it's making sure no jobs triggered by the same branch/tag run together — does it also ensure if a job is running for stable (i.e. upon push to tag), a new job for nightly (i.e. upon push to dev) will wait for the stable job to finish before running?

Reply:
Hm... you 're right. Just modified the concurrency group to the workflow level: https://github.com/ntampakas/tlsn/commit/445c1afa11482d39bc7c20a181fd22d3da2971d7